src/include/OpenImageIO/imagebufalgo.h

```diff
@@ -2163,6 +2163,9 @@ bool OIIO_API repremult (ImageBuf &dst, const ImageBuf &src,
 ///
 ///     The order the color filter array elements are arranged in, pattern-specific.
 ///
+///   - "white-balance" : float[3] or float[4] (default: {1.0, 1.0, 1.0, 1.0})
+///
+///     Optional white-balancing weights. Can contain either three (RGB), or four (RGGB) values.
```

Comment:
Is there a use case for specifying two green values?

Reply:
Yes, I have seen some images recently with the greens in red and blue rows having distinctly different levels. Sony A99, I think.

Reply:
> Is there a use case for specifying two green values?

RAW_PENTAX_K200D.PEF in the test suite has 2 different weights for the green sub-channels